codespan-reporting/CHANGELOG.md

```diff
@@ -25,6 +25,8 @@ and a [contributing guide](https://github.com/brendanzab/codespan/blob/master/CO
 
 ### Fixed
 
+-   Implementations of the `Files` trait are simplified by automatically including the
+    end of file byte index as a "line end".
```

Comment:
Wondering if this should be moved to 'changed'. Along with a description of how to update existing implementations of `Files`?

Reply:
Guess I don't know if this is an improvement for end-users or not, or if it's even more confusing. Maybe @Marwes could chime in?